include/language/en_us.lang.php

```diff
@@ -464,6 +464,7 @@
         'Accounts' => 'Account',
         'Contacts' => 'Contact',
         'Opportunities' => 'Opportunity',
+        'Campaigns' => 'Campaign',
```

Comment:
Please use singular or plural consistency for key and value (as there are other strings for campaigns/campaign)